requirements.in

```diff
@@ -6,7 +6,7 @@ click==6.7
 fanstatic==0.12
 Flask==0.12.4
 Flask-Babel==0.11.2
-Jinja2==2.8
+Jinja2>=2.8.1
```

Comment:
Can you pin it to this version? Otherwise people will get 2.10 which might be incompatible